docs/checks/docs_link_checker.py

```diff
@@ -144,7 +144,7 @@ def _get_relative_path(self, file: str, path: str) -> str:
         return os.path.join(directory, self._get_os_path(path))  # noqa: PTH118
 
     def _get_docroot_path(self, path: str) -> str:
-        return os.path.join(self._docs_root, self._get_os_path(path))  # noqa: PTH118
```

Comment:
Due to the repeating nature of the `/docs/docusaurus/docs/` path,  the `os.path.join()` function was getting tripped up and not appending the paths. A string concatenation works

Reply:
I wonder if this is more of a configuration issue; the path is provided as a runtime parameter